tiledb/sm/query/query.h

```diff
@@ -449,7 +449,7 @@ class Query {
    */
   bool has_results() const;
 
-  /** Initializes the query. */
+  /** Initializes the query (on submission, called from submit() et al.). */
```

Comment:
We typically do not doc where we expect a routine to be used, because it often changes in the future. Remove this addition to the comment.

(There are exceptions, such as annotating that a function is public because we need it to be accessible to unit tests or serialization, etc).

Reply:
Done